ROVU/Core/src/model/Node.java

```diff
@@ -73,7 +73,6 @@ public Node(boolean wall, boolean wifi, boolean eat, int id, Point p){
         this.wall = wall;
         this.wifi = wifi;
         this.eating = eat;
-        this.room = room;
```

Comment:
Don't we need this in the future?

Reply:
@nicket97 You defined `Node`, what do you think? Also, there's the problem of having `Node` defined in the package `Robot`, and the package `Model`. Are they two different types of nodes??

Reply:
`Node.java` has been rewritten, and I've merged in those changes to this branch now.